engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetViewportToSizeOf.java

```diff
@@ -35,12 +34,17 @@
     private static SetViewportToSizeOf defaultInstance;
 
     private BaseFBOsManager fboManager;
-    private SetViewportToSizeOfTask task;
     private ResourceUrn fboName;
+    private int fboWidth;
+    private int fboHeight;
 
     public SetViewportToSizeOf(ResourceUrn fboName, BaseFBOsManager frameBuffersManager) {
         this.fboManager = frameBuffersManager;
         this.fboName = fboName;
+
+        // Cheeky way to initialise fboWidth, fboHeight
```

Comment:
This line used to be beside update() for good reason. Now it seems to related to both lines below. I think something similar happens in another class, earlier.